bash/lz390

```diff
@@ -25,6 +25,7 @@ fi
 java -classpath ${jardir}/z390.jar -Xrs $J2SEOPTIONS lz390 $1 $2 $3 $4 $5 $6 $7 $8 $9
 
 rc=$?
+# rc = 1 for Java issues; all other non-zero values for linker issues
```

Comment:
Were you going to print something based on the rc.. or just add a comment in case someone looks through the script?